src/test/java/org/broadinstitute/hellbender/engine/ReadsDataSourceUnitTest.java

```diff
@@ -554,6 +563,20 @@ private File getFileWithAddedContig(
         };
     }
 
+    @DataProvider(name="cloudXorTestData")
+    public Object[][] cloudXorTestData() {
```

Comment:
Move this `DataProvider` next to the `testCloudBamWithCustomReaderFactoryAndWrappers()` test case